docs/time/index.rst

```diff
@@ -485,38 +531,38 @@ The allowed |Time| arguments to create a time object are listed below:
 **val2** : numpy ndarray, list, str, or number; optional
     Data to initialize table.
 **format** : str, optional
-    Format of input value(s)
+    Format of input value(s).
 **scale** : str, optional
-    Time scale of input value(s)
+    Time scale of input value(s).
 **precision** : int between 0 and 9 inclusive
-    Decimal precision when outputting seconds as floating point
+    Decimal precision when outputting seconds as floating point.
 **in_subfmt** : str
-    Unix glob to select subformats for parsing input times
+    Unix glob to select subformats for parsing input times.
 **out_subfmt** : str
-    Unix glob to select subformat for outputting times
+    Unix glob to select subformat for output times.
 **location** : |EarthLocation| or tuple, optional
-    If a tuple, 3 |Quantity| items with length units for geocentric coordinates,
-    or a longitude, latitude, and optional height for geodetic coordinates.
-    Can be a single location, or one for each input time.
+    If a tuple, three |Quantity| items with length units for geocentric
+    coordinates, or a longitude, latitude, and optional height for geodetic
+    coordinates. Can be a single location, or one for each input time.
 
 val
 ^^^
 
-The ``val`` argument  specifies the input time or times and
-can be a single string or number, or it can be a Python list or `numpy` array
-of strings or numbers. To initialize a |Time| object based on a specified time,
-it *must* be present.  If ``val`` is absent (or `None`), the |Time| object will
-be created for the time corresponding to the instant the object is created.
+The ``val`` argument specifies the input time or times and can be a single
+string or number, or it can be a Python list or ```numpy`` array of strings or
+numbers. To initialize a |Time| object based on a specified time, it *must* be
+present. If ``val`` is absent (or `None`), the |Time| object will be created for
```

Comment:
The last sentence is not true (for a long time, I fear). I'd just remove "If ``val`` is absent..."